.semaphore/versions.rb

```diff
@@ -0,0 +1,12 @@
+# DO NOT EDIT
+ # This is a file downloaded by the `script/generate_ci_matrix` task.
+ # Update it in appsignal-elixir, then run script/generate_ci_matrix to
+ # redownload it.
```

Comment:
```suggestion
# This is a file downloaded by the `script/generate_ci_matrix` task.
# Update it in appsignal-elixir, then run script/generate_ci_matrix to
# redownload it.
```
Running `script/generate_ci_matrix` with the suggestions below will remove these leading spaces.